Dockerfile

```diff
@@ -0,0 +1,18 @@
+FROM library/node:6.3
+
+WORKDIR /src
+
+COPY package.json .
```

Comment:
We could do it for the bower install, but if we leave it bundled with the tools/prepare.sh it won't work because prepare.sh requires more of the environment to be present.